numcodecs/tests/test_pcodec.py

```diff
@@ -57,14 +61,23 @@ def test_config():
 
 
 def test_invalid_config_error():
-    codec = PCodec(mode_spec='bogus')
+    for param in ["mode_spec", "delta_spec", "paging_spec"]:
```

Comment:
Might be a bit nicer to use [pytest to parametrize](https://docs.pytest.org/en/stable/example/parametrize.html) the test instead of having this for loop?